### ISSUE TITLE
How to get the FG-ZS-SBIR datasets label infomation 

### ISSUE THREAD

**Issue by develop-productivity** [mainaksingha01/SpLIP#2]
Hi, Thank you for such an outstanding work.
Regarding the FG-ZS-SBIR test indicator, I would like to know how to obtain relevant dataset annotations to obtain fine-grained computing support. If possible, I would like to cite your paper to prove your help to the sketch community.

Thanks

**Comment by mainaksingha01** [comment]
Hi, thanks for your interest in our work. In the FG-ZS-SBIR set up, we have used the Sketchy dataset only i.e. basic one without the 'ext' files. There are at least around 5 to 6 corresponding sketches for each photo with similar filenames in a particular class, that we have taken as anchor and positive instances, where rest of the photos for the same class are considered as negative instances.

**Comment by develop-productivity** [comment]
> Hi, thanks for your interest in our work. In the FG-ZS-SBIR set up, we have used the Sketchy dataset only i.e. basic one without the 'ext' files. There are at least around 5 to 6 corresponding sketches for each photo with similar filenames in a particular class, that we have taken as anchor and positive instances, where rest of the photos for the same class are considered as negative instances.

Yes, there are a lot of sketchs for each photo with similar filenames in a particular class. Thanks.
